src/compiler/commandLineParser.ts

```diff
@@ -262,7 +262,9 @@ namespace ts {
                 "es3": ScriptTarget.ES3,
                 "es5": ScriptTarget.ES5,
                 "es6": ScriptTarget.ES6,
+                "es8": ScriptTarget.ES8,

```

Comment:
you also need to update `getDefaultLibFileName` to inject the right library based on the target.